bank.engineering.yaml

```diff
@@ -3,3 +3,16 @@ changelog:
   ttl: 1
   type: CNAME
   value: headwayapp.co.
+_acme-challenge.curl:
+  ttl: 1
+  type: CNAME
+  value: c82b414cd89d32fd90685ead._acme.deno.dev.
+curl:
+  ttl: 1
+  type: A
+  value: 34.120.54.55
+curl:
+  ttl: 1
+  type: AAAA
+  value: 2600:1901:0:6d85::

```

Comment:
```suggestion
curl:
  - ttl: 1
    type: A
    value: 34.120.54.55
  - ttl: 1
    type: AAAA
    value: 2600:1901:0:6d85::

```

Reply:
so the mapping was fine - i thought that's what the linter was complaining about.

Reply:
my dumb arse commited before merging your changes -yay

Reply:
@reesericci you also have to nest both records under the same `curl` key